### DISCUSSION TITLE
Encryption with Termux - package or script to encrypt/decrypt directories

### DISCUSSION THREAD

**Discussion by francwalter** [termux/termux-packages#17430] [Q&A] [answered]
Hallo

is there a package for Termux for encryption, beside of gpg?
I need a command to encrypt whole folders, not only a file. gpg can not encrypt directories.

I want to use it in the Tasker Plugin, because Tasker has no encryption in the Google Play version.

I asked and commented that already in the [Tasker's Google Groups](https://groups.google.com/g/tasker/c/n11Ck8rKPCo/m/OHCLg-iGAAAJ), but not much of interest about it there (as if it is not useful to encrypt files on a phone).

Thanks for hints.
frank

**Comment by tareksander** [comment]
You can pack a folder into a file with `tar` and then encrypt it.

**Comment by sylirre** [comment, marked as answer]
`git` + `git-crypt`, if you really look for directories encryption. Other (non-root) tools that we have operate on files only: age, ccrypt, scrypt and few others.

If your device is rooted, then you can try `encfs` package.

**Reply by francwalter** [reply]
` git-crypt enables transparent encryption and decryption of files in a git repository.`
But I dont want to encrpyt git repos, just plain directories. I dont see, how I can realize that with git-crypt. Without creating a git repo.
